src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
+        "        #\n"
+        "        # You can create a signature file using `gpg` like so:\n"
+        "        #\n"
+        "        #     echo -n \"I am the author of the Sandstorm.io app with the following ID: <app-id>\" | gpg --sign > pgp-signature\n"
+        "        #\n"
+        "        # To learn how to set up gpg, visit Keybase (https://keybase.io) -- they have excellent\n"
+        "        # documentation and tools. Moreover, if you create a Keybase account for your key and follow\n"
+        "        # Keybase's instructions to link it to other social accounts (like your Github account), then\n"
+        "        # the Sandstorm app install flow and app market will present this information to the user as\n"
+        "        # \"verified\".\n"
+        "\n"
+        "        #upstreamAuthor = \"Example App Team\",\n"
+        "        # Name of the original primary author of this app, if it is different from the person who\n"
+        "        # produced the Sandstorm package. Setting this implies that the author connected to the PGP\n"
+        "        # signature only \"packaged\" the app for Sandstorm, rather than developing the app.\n"
+        "      ),\n"
+        "      pgpKeyring = embed \"path/to/pgp-keyring\",\n"
+        "      # A keyring in GPG keyring format containing all public keys needed to verify PGP signatures in\n"
+        "      # this manifest (as of this writing, there is only one: `author.pgpSignature`).\n"
+        "      #\n"
+        "      # To generate a keyring containing just your public key, do:\n"
+        "      #\n"
+        "      #     gpg --export <key-id> > keyring\n"
+        "      #\n"
+        "      # Where `<key-id>` is a PGP key ID or email address associated with the key.\n"
+        "\n"
+        "      description = (defaultText = embed \"path/to/description.md\"),\n"
+        "      # The app's description description in Github-flavored Markdown format, to be displayed e.g.\n"
+        "      # in an app store. Note that the Markdown is not permitted to cotnain HTML nor image tags (but\n"
+        "      # you can include a list of screenshots separately).\n"
+        "\n"
+        "      shortDescription = (defaultText = \"one-to-three words\"),\n"
+        "      # A very short (one-to-three words) description of what the app does. For example,\n"
+        "      # \"Document editor\", or \"Notetaking\", or \"Email client\". This will be displayed under the app\n"
+        "      # title in the grid view in the app market.\n"
+        "\n"
+        "      screenshots = [\n"
+        "        # Screenshots to use for marketing purposes.  Examples below.\n"
+        "\n"
+        "        #(width = 746, height = 795, jpeg = embed \"path/to/screenshot-1.jpeg\"),\n"
+        "        #(width = 640, height = 480, png = embed \"path/to/screenshot-2.png\"),\n"
+        "      ],\n"
+        "      changeLog = (defaultText = embed \"path/to/sandstorm-specific/changelog.md\"),\n"
```

Comment:
Can we auto-generate an empty `app-market-changelog.md` like how GitHub does `README.md` in fresh git repositories? (Admittedly I'm suggesting we do it by default whereas for them it's an option.)